packages/plugins/src/foundry/utils/trim.ts

```diff
@@ -0,0 +1,143 @@
+import { BuildInfo } from '@sphinx-labs/core'
+
+export const trimObjectToType = <T>(obj: any, typeTemplate: T): T => {
```

Comment:
This function is used to trim down the build info file to exactly the fields we specify in our type. To do this, we use a template object which contains all the fields we want. We cannot use the typescript type itself since that doesn't exist at runtime. So, if we want to update the `BuildInfo` type, we also need to update the template here.